client/packages.go

```diff
@@ -91,10 +91,23 @@ type FindOptions struct {
 
 var ErrNoSnapsInstalled = errors.New("no snaps installed")
 
+type ListOptions struct {
+	All bool
+}
+
 // List returns the list of all snaps installed on the system
 // with names in the given list; if the list is empty, all snaps.
-func (client *Client) List(names []string) ([]*Snap, error) {
-	snaps, _, err := client.snapsFromPath("/v2/snaps", nil)
+func (client *Client) List(names []string, opts *ListOptions) ([]*Snap, error) {
+	if opts == nil {
+		opts = &ListOptions{}
+	}
+
+	q := make(url.Values)
+	if opts.All {
+		q.Add("all", "1")
```

Comment:
Looking at the implementation of /v2/find, we have "select=refresh" and "select=private" there, for example, which change the selection of snaps the operation applies to. If we want to follow suit, that might be "select=all", and defaults to "enabled" (implied when empty)?

Alternatively, if select was a bad plan we might sunset it at some point and move it to individual flags instead. But it'd be nice to have lookalike APIs on those two endpoints.